test/sequential/test-module-loading.js

```diff
@@ -127,10 +127,6 @@ assert.equal(require('../fixtures/registerExt2').custom, 'passed');
 assert.equal(require('../fixtures/foo').foo, 'ok',
              'require module with no extension');
 
-assert.throws(function() {
```

Comment:
My thinking is this PR changes the behavior of the code and there ought to be a test for that change. So a test that the property has, in fact, been successfully removed seems appropriate. But if you think it's overkill, I'm OK without it. (I'll edit my comments above to indicate they are nits.)